### PR TITLE
♻️ `director-v2` fixing mypy issues part 1/3

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

Since there are lots of changes to be made in this service. Starting with a first round.

## Related issue/s

- https://github.com/ITISFoundation/osparc-simcore/issues/4071

### CODE REVIEW

File: services/director-v2/src/simcore_service_director_v2/models/schemas/clusters.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ class WorkerMetrics(BaseModel):
     task_counts: TaskCounts = Field(..., description="task details")
 
 
-AvailableResources = DictModel[str, PositiveFloat]
+AvailableResources: TypeAlias = DictModel[str, PositiveFloat]
```
Comment (pcrespov) [inline services/director-v2/src/simcore_service_director_v2/models/schemas/clusters.py:42]: THOUGHT: I thought we wanted to get rid of these ``DictModel``s  by using directly ``dict[str, PositiveFloat]``.

**Review by pcrespov — APPROVED** [review]
:tada: Cool! thx

**Review by sanderegg — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/director-v2/src/simcore_service_director_v2/api/dependencies/rabbitmq.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from fastapi import Request
-
-from ...modules.rabbitmq import RabbitMQClient
+from servicelib.rabbitmq import RabbitMQClient
 
 
 def get_rabbitmq_client(request: Request) -> RabbitMQClient:
+    assert type(request.app.state.rabbitmq_client) == RabbitMQClient  # nosec
```
[the remark this answers is not in the document]
Comment (GitHK) [reply services/director-v2/src/simcore_service_director_v2/api/dependencies/rabbitmq.py:6]: I do not agree with you here. Consider the following code.

```python
class A:
    pass


class B(A):
    pass


a = A()
b = B()

assert type(b) == B
assert type(b) != A        # obviously

assert isinstance(b, B)
assert isinstance(b, A)    # correct but not what we want here
```

File: services/director-v2/src/simcore_service_director_v2/models/schemas/clusters.py (hunk 1 above)
Comment (GitHK) [reply services/director-v2/src/simcore_service_director_v2/models/schemas/clusters.py:42]: If we want to do that, I guess adding a case will remind us to deprecate it

**Pull request merged**